high_card.rb

```diff
@@ -1,5 +1,121 @@
+require "pry"
+
 # Use these two arrays to generate a deck of cards.
 ranks = ["A", 2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K" ]
 suits = [ "hearts", "spades", "clubs", "diamonds" ]
 
-players = []
+the_deck = ranks.product(suits)
+
+class Game
+	attr_accessor :players
+	def initialize(deck)
+		@deck = deck
+		@players = []
+	end
+	def add_players(user)
+		@players.push(user)
+	end
+	def deal_players
+		@players.each do |x|
+			x.deal(@deck)
+		end
+	end
+	def clear_cards
+		@players.each do |x|
+			x.new_cards
+		end
+	end
+	def rank_the_deck
+		@deck.each do |y|
+			case y[0]
+			when "A"
+				y.push(14)
```

Comment:
Perhaps there's a way we could use each card's index position within `ranks` to assign its value, rather than have to use additional conditional statements to deal with the letter cards...